# Use this file to specify parameters, that do not have default values.
# YAML Files can also be helpful if you want to save parameters for different use-cases.
# In any way: Default values should go into the .if file!
#
# To generate a yaml file, use "rosrun rosinterface_handler generate_yaml /path/to/PathGenerator.if"
#
# Use 'key: value' pairs, e.g.
# string: 'foo'
# integer: 1234
# float: 1234.5
# boolean: true
# vector: [1.0, 2.0, 3.4]
# map: {"a": "b", "c": "d"}

verbosity: debug

frame_id_map: world
frame_id_vehicle: vehicle_front_axle
frame_id_camera: camera_top
frame_id_stargazer: stargazer
#this has to be synchronized with point distance in path_publisher
output_distance: 0.02
input_distance: 0.01
